files/benchmark_job.yml

```diff
@@ -43,6 +43,10 @@
                          name: branch
                          default: 'master'
                          description: 'The benchmark_harness branch to use'
+                 - string:
+                         name: script_branch
+                         default: 'master'
+                         description: 'The hpc_lab_setup branch to use'
```

Comment:
I wouldn't use the repository name, here, as it can change.
This is about the Ansible playbook branch, so you can just say that.
Or even call it "automation scripts" if we want to be extra generic.